environment.yml

```diff
@@ -0,0 +1,16 @@
+name: fre-cli
+channels:
+  - defaults
+  - conda-forge
+  - noaa-gfdl
+dependencies:
+  - python
+  - click
+  - pyyaml
+  - pylint
+  - jsonschema
+  - noaa-gfdl::intakebuilder
+  - conda-forge::cylc-flow
+  - conda-forge::cylc-rose
+  - conda-forge::metomi-rose
+  - conda-forge::cmor
```

Comment:
I don't disagree with an environment.yaml here but is it a duplicate of meta.yaml? We might forget to update this one. Is there something here that's not in the meta.yaml?

Reply:
essentially, yes.

my thinking here is that `environment.yml` builds a bare environment into which the `fre-cli` can be pip-installed as the sole instance of `fre`. That being said, from in-person conversations with @bcc2761 , this doesn't seem to gain us anything over `pip install`'ing `fre-cli` on top of the conda package one and explicitly invoking `fre` using the full path to the conda environment's executable